src/stratigraphy/util/depthcolumn.py

```diff
@@ -207,10 +209,19 @@ def strictly_contains(self, other: BoundaryDepthColumn):
         )
 
     def depth_intervals(self) -> list[BoundaryInterval]:
+        """Creates a list of depth intervals from the depth column entries.
+
+        The first depth interval is open-ended.
+
+        Returns:
+            list[BoundaryInterval]: A list of depth intervals.
+        """
         depth_intervals = [BoundaryInterval(None, self.entries[0])]
         for i in range(len(self.entries) - 1):
             depth_intervals.append(BoundaryInterval(self.entries[i], self.entries[i + 1]))
-        depth_intervals.append(BoundaryInterval(self.entries[len(self.entries) - 1], None))
+        depth_intervals.append(
+            BoundaryInterval(self.entries[len(self.entries) - 1], None)
+        )  # even though no open ended intervals are allowed, they are still useful for matching
```

Comment:
Add: "especially for documents where the material description rectangle is too tall (and includes additional lines below the actual material descriptions)"